tools/serverpod_cli/lib/src/commands/create_migration.dart

```diff
@@ -61,6 +84,18 @@ class CreateMigrationCommand extends ServerpodCommand<CreateMigrationOption> {
   ) async {
     bool force = commandConfig.value(CreateMigrationOption.force);
     String? tag = commandConfig.optionalValue(CreateMigrationOption.tag);
+    bool check = commandConfig.value(CreateMigrationOption.check);
+    bool empty = commandConfig.value(CreateMigrationOption.empty);
+
+    // Validate that check and empty flags are not used together
+    if (check && empty) {
+      log.error(
+        'Cannot use both --check and --empty flags together. '
+        'The --check flag checks for changes without creating migrations, '
+        'while --empty creates an empty migration.',
```

Comment:
Suggestion(NIT): We can avoid the repeated command description, since it is already present in the help. The first line is already clear for the user.

```suggestion
        'Cannot use both --check and --empty flags together. ',
```